### DISCUSSION TITLE
"'fzf' is not a valid executable" in v14

### DISCUSSION THREAD

**Discussion by bbaldino** [LazyVim/LazyVim#5046] [Q&A] [answered]
Just upgraded to v14 and I'm getting this error:

`[Fzf-lua] 'fzf' is not a valid executable, aborting. Please make sure 'fzf' is in installed.`

In the past I've seen this sort of thing from an issue during the install, so I uninstalled fzf-lua and then re-installed it but that didn't seem to address it: it does re-install but I still get the same error.  Do I need to install fzf separately? Or should LazyVim take care of installing the binary? Just trying to make sure I don't introduce a conflict.

Thanks!

**Comment by folke** [comment, marked as answer]
You need to install fzf. Or  enable telescope if you don't want to use fzf-lua

**Reply by bbaldino** [reply]
Got it, thanks!

**Reply by SargisPlusPlus** [reply]
same, thank you

**Comment by SargisPlusPlus** [comment]
For anyone else having same issue on Mac, run `brew install fzf` 
You won't regret.
